code/street-banner/street-banner.js

```diff
@@ -215,6 +215,13 @@ $(document).on('knack-view-render.view_3032', function(event, page) {
   // create large button on the home page
     bigButton('account-management', 'view_3032', "https://atd.knack.com/street-banners#account-management/", "users", "Account Management");
 });
+
+//SMO App link button for the Admin tab
+$(document).on('knack-view-render.view_3689', function(event, page) {
+  // create large button on the home page
+    bigButton('smart-mobility', 'view_3689', "https://atd.knack.com/smart-mobility#home/", "street-view", "Smart Mobility Office");
```

Comment:
I'm wondering if this button should open up in another window rather than going directly to the other URL.

By adding `true `as the last argument in the bigButton function, it sets the target attribute of the button's anchor tag to `_blank`, which opens the link in a new window or tab.

`bigButton('smart-mobility', 'view_3689, "https://atd.knack.com/smart-mobility#home/", "street-view", "Smart Mobility Office", true);`

Reply:
In the SMD Tracker, the link to the Street Banner app takes then using their existing tab, it doesn't create a separate one. I think the assumption is you're accepting to navigating away into a new application, no?

Reply:
if that's the case, then disregard my suggestion to open in a separate tab.